knoten/__init__.py

```diff
@@ -9,7 +9,12 @@
 from csmapi import csmapi
 
 # Register the usgscam plugin with the csmapi
+libusgscsm_path = find_library('usgscsm')
 
-lib = ctypes.CDLL(find_library('usgscsm'))
-if not lib:
+if not libusgscsm_path:
+    warnings.warn('libusgscsm not installed, unable to load shared library.')
+
+libusgscsm = ctypes.CDLL(libusgscsm_path)
+
+if not libusgscsm._name:
     warnings.warn('Unable to load usgscsm shared library')
```

Comment:
Should this warning be removed?

Reply:
I imagine it's still cases where usgscsm is installed but fails to load? (e.g.off chance that usgscsm was installed manually but some dependencies like libcsm were not installed)